ffs/arbitration.md

```diff
@@ -0,0 +1,114 @@
+
+## 10. Proof
+
+If a party makes a claim, it is their responsibility to support it. A screenshot is reasonable evidence if it is not manipulated. Giving details related to an event is plausible evidence if the details can be independently confirmed. Payments can be proved with the submission of Transaction IDs, Transaction Keys, Payment IDs, public address, blockchain hash and block height data. Verifying a signed message with a private key can prove ownership of a public address. Cryptographically timestamping can prove information existed at a given point in time. The tribunal may examine metadata embedded in documents and electronic communications in order to assess the evidential value of such information.
+
+## 11. Conflicts of Interest
+
+A conflict of interest arises if a party has one of the following relationships to an arbitrator:
+
+- Business partner
+- Employer or employee
+- Stakeholder
+- Close friend or relative
+
+All parties must reveal any evidence of a conflict of interest as soon as possible before or during a dispute. In assessing conflict of interests, arbitrators shall have regard for the [IBA Guidelines on Conflicts of Interest](https://www.ibanet.org/Document/Default.aspx?DocumentUid=E2FE5E72-EB14-4BBA-B10D-D33DAFEE8918).
+
+## 12. Status
+
+There is no centralized authority, entity, or institution that controls, dictates, regulates, or makes decisions regarding the relationship between the parties and the arbitrators. The arbitrators are volunteers and parties directly interact with the arbitrators by virtue of an arbitration agreement and these Rules. Nothing contained in the Rules create an employer and employee, partnership, joint venture, or any other fiduciary relationship. The arbitrators are not agents for, or act on behalf of, any of the parties.
+
+## 13. Place of Arbitration
+
+The place of arbitration shall be Geneva, Switzerland. The award shall be deemed to have been made at the place of arbitration. The tribunal may conduct the arbitration at any location they consider appropriate, including, online through videoconferencing or any other electronic communication mediums.
+
+## 14. Limitation of Liability
+
+Save for intentional wrongdoing, the parties waive, to the fullest extent permitted under the applicable law, any claim against the arbitrators and repository contributors based on any act or omission in connection with the arbitration or the development of these Rules.
+
+## 15. Power to Determine Authority
+
+The tribunal has the power to determine if they have the authority to arbitrate the dispute between the parties even if the contract establishing the relationship between the parties is alleged to be invalid or non-existent.
+
+## 16. Involvement of National Courts
+
+No court shall intervene in matters governed by the arbitration agreement except where so provided by law.
```

Comment:
>No court shall intervene in matters governed by the arbitration agreement except where so provided by law.

Which law and where?

Reply:
There is a general principle that when parties have agreed to arbitration, [courts cannot get involved in the dispute](https://www.trans-lex.org/450900/_/uncitral-model-law-on-international-commercial-arbitration/#head_7). However, in many jurisdictions, a party may request an interim court order to stop a party from doing something that may cause irreparable harm to the other while a dispute is being handled by an arbitral tribunal.